setup.cfg

```diff
@@ -26,6 +26,7 @@ packages = find:
 install_requires =
     attrs
     cffi>=1.0.0
+    Pillow
```

Comment:
We should probably use a setuptools extra here, in case folks don't actually need Pillow.

The way you do that in a setup.cfg is:

```ini
[options.extras_require]
pillow =
    Pillow
```

which then in the installation, someone should run `pip install avif[pillow]` instead of just `pip install avif` if they want the pillow support as well.